.github/workflows/php-unit-tests.yml

```
@@ -0,0 +1,79 @@
+name: PHP Unit Tests
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "**.php"
+      - ./composer.json
+      - ./composer.lock
+      - ./.github/workflows/php-unit-tests.yml
+      - ./.github/actions/prepare-php/action.yml
+  pull_request:
+    paths:
+      - "**.php"
+      - ./composer.json
+      - ./composer.lock
+      - ./.github/workflows/php-unit-tests.yml
+      - ./.github/actions/prepare-php/action.yml
```

I haven't tested it, but looking at https://github.com/woocommerce/google-listings-and-ads/pull/1541/files this seems to be 
```suggestion
      - "**.php"
      - composer.json
      - composer.lock
      - .github/workflows/php-unit-tests.yml
      - .github/actions/prepare-php/action.yml
  pull_request:
    paths:
      - "**.php"
      - composer.json
      - composer.lock
      - .github/workflows/php-unit-tests.yml
      - .github/actions/prepare-php/action.yml
```

---

It works both ways, but your version is cleaner, thanks.